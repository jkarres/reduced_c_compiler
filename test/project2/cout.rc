// testing cout

function : int i() {
  cout << 3 << " " << "in i()" << endl;
  return -123;
}

function : float f() {
  cout << 4 << " " << "in f()" << endl;
  return 1.2;
}

function : bool b() {
  cout << 5 << " " << "in b()" << endl;
  return false;
}

function : void main() {
  cout << 6 << " " << i() << endl << f() << endl << b() << endl;
}
