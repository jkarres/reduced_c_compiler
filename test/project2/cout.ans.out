6 3 in i()
-123
4 in f()
1.20
5 in b()
false
